modules/microshift-storage-creating-vol-snapshot.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// microshift/creating-volume-snapshots-microshift.adoc
+
+:_content-type: PROCEDURE
+[id="creating-a-volume-snapshotting_{context}"]
+= Creating a volume snapshot
+
+To create a snapshot of a {product-title} storage volume, you must first configure {op-system-ostree} and the cluster. In the following example procedure, the pod that the source volume is mounted to is deleted. Deleting the pod prevents data continuing to be written to it during snapshot creation. If the pod is managed with a replication controller, meaning that a new pod is instantly created when one is deleted, temporarily scale the replica count to zero instead of deleting the pod.
+
+[NOTE]
+====
+Volume snapshots are located on the same devices as the original data. To use the volume snapshots as backups, move the snapshots to a secure location.
+====
+
+.Prerequisites
+
+. User has root access to a {product-title} cluster.
+. A {product-title} cluster is running.
+. A device class defines an LVM thin-pool.
+. A `volumeSnapshotClass` specifies `driver: topolvm.io`.
+. Any workload attached to the source PVC is paused or deleted. This helps avoid data corruption.
+
+[IMPORTANT]
+====
+All writes to the volume must be halted while you are creating the snapshot. If you do not halt writes, your data might be corrupted.
+====
+
+.Procedure
+
+. Delete the pod to ensure that no data is written to the volume during snapshotting by running the following command:
++
+[source, terminal]
+----
+$ oc delete my-pod
+----
+
+. Optional: If the pod is managed with a replication controller, meaning that a new pod is instantly created when one is deleted, temporarily scale the replica count to zero instead of deleting the pod.
+
+. Once the pod has been deleted, run a command similar to the example that follows. Insert your own configuration details.
++
+.Example snapshot configuration
++
+[source, terminal]
+----
+# oc apply -f <<EOF
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot <1>
+metadata:
+  name: <my-snap> <2>
+spec:
+  volumeSnapshotClassName: topolvm-snapclass <3>
+  source:
+    persistentVolumeClaimName: test-claim-thin <4>
+EOF
+----
+[.small]
+<1> Create a `VolumeSnapshot` object.
+<2> The name that you specify for the snapshot.
+<3> Specify the desired `VolumeSnapshotClass`.
+<4> Specify either `.persistentVolumeClaimName` or `volumeSnapshotContentName`. In this example, we are creating a snapshot from a PVC. The value `test-claim-thin` is the source PVC.
```

Comment:
```suggestion
<4> Specify either `persistentVolumeClaimName` or `volumeSnapshotContentName`. In this example, we are creating a snapshot from a PVC. The value `test-claim-thin` is the source PVC.
```

I think the `.` was just a typo, but it makes it look like we're giving inconsistent syntax.